x-pack/plugins/aiops/server/routes/queries/fetch_change_point_p_values.ts

```diff
@@ -23,7 +23,8 @@ import { getRequestBase } from './get_request_base';
 
 export const getChangePointRequest = (
   params: AiopsExplainLogRateSpikesSchema,
-  fieldName: string
+  fieldName: string,
+  sampleProbability: number = 1
```

Comment:
Minor suggestion: I think it would be helpful to note here that it means no sampling at all when sample probability = 1. Might be obvious, but doesn't hurt to add a comment for clarity.

Reply:
I added comments where this argument is used. Additionally, I updated the logic to only apply the wrapping random sampler aggregation when `sampleProbability` is smaller than 1 in [209f716](https://github.com/elastic/kibana/pull/143977/commits/209f7165fca4c7328c7b4994b40cde580d6baf5f).